config/routes.rb

```diff
@@ -37,4 +37,4 @@
   resources :deck_subscriptions, only: [:create, :edit, :show, :update] do
     resources :study_sessions, only: [ :create, :update, :show ]
   end
-end
\ No newline at end of file
+end
```

Comment:
At first I thought this was annoying and went to remove the new line added by atom, but then I began reading why atom adds new lines: http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795
and I am now convinced its a good idea to keep it in.